NumLib/ODESolver/TimeDiscretization.h

```diff
@@ -200,14 +213,32 @@ class TimeDiscretization
      */
     virtual bool needsPreload() const { return false; }
     //! @}
+
+protected:
+    std::unique_ptr<GlobalVector> _dx; ///< Used to store \f$ u_{n+1}-u_{n}\f$.
```

Comment:
If there are thousands of time steps, locating and releasing the vector could slow down the computation.